tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala

```diff
@@ -0,0 +1,64 @@
+package org.scalafmt.internal
+
+import scala.meta.testkit._
+
+import scala.collection.GenIterable
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.meta._
+import scala.util.control.NonFatal
+
+import java.util.concurrent.CopyOnWriteArrayList
+import java.util.concurrent.atomic.AtomicInteger
+
+object SyntaxAnalysis {
```

Comment:
Can we add `// + scalameta deviation` `// - scalameta deviation` comments to distinguish what has changed?

Reply:
I removed a println